addon/mirage-graphql/schema.graphql

```diff
@@ -2138,6 +2176,23 @@ type SaveDocumentFloatAnswerPayload {
   clientMutationId: String
 }
 
+input SaveDocumentFormAnswerInput {
+  question: ID!
+  document: ID!
+  meta: JSONString
+
+  """
+  Document IDs representing the content of the form.
+  """
+  value: ID!
+  clientMutationId: String
+}
+
+type SaveDocumentFormAnswerPayload {
+  answer: Answer
+  clientMutationId: String
+}
+
 input SaveDocumentInput {
   id: String
   form: ID!
```

Comment:
Did you run `yarn update-schema` with the **latest** container? This seems like a deprecated version...